Src/UIComponents/Buttons/Buttons.ts

```diff
@@ -0,0 +1,31 @@
+function generateButton(label:string, onClick:Function, infill:boolean) :HTMLButtonElement {
+   
+    const button:HTMLButtonElement = document.createElement('button');
+    button.className = 'button';
+    button.innerText = label;
+
+    if(infill){
+        button.style.backgroundColor = "#DA1B36";
+        button.style.color = "white";
+    } else {
+        button.style.backgroundColor = "white";
+        button.style.color = "#DA1B36";
+    }
```

Comment:
I suggest to use ternary operator, for example:
button.style.backgroundColor = infill ?  ""#12321" : "white"

Reply:
definitely, 5 lines less :) done